### PR TITLE
Feature 79 restaurant list voznyarskyy

### PR DESCRIPTION
**Pull request by VozNick**

### CODE REVIEW

File: server/src/main/java/com/lunchtime/models/RestaurantVisit.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.lunchtime.models;
```
Comment (vitaliimak) [inline server/src/main/java/com/lunchtime/models/RestaurantVisit.java:1]: Why you create this model? We have restaurant model. A lot of users (persons) use our application, will you plane to create separate RestaurantVisit entity for every user?
Is your task need this functionality?

File: server/src/main/java/com/lunchtime/implementation/RestaurantServiceImplement.java [hunk 2]
```diff
@@ -40,6 +40,36 @@ public Restaurant save(Restaurant restaurant) {
...
+    }
+
+    @Override
+    public Page<Restaurant> yourPastVisits(Pageable pageable) {
+        return null;
+    }
+
+    @Override
+    public Page<Restaurant> chooseType(Pageable pageable) {
+        return null;
```
Comment (vitaliimak) [inline server/src/main/java/com/lunchtime/implementation/RestaurantServiceImplement.java:70]: If this code is useless you shouldn't merge it in dev branch

File: client/src/components/info/restaurants.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const sortBy = ['Relevance', 'Distance', 'Your match'];
+const city = ['Вінницька область', 'Волинська область', 'Дніпропетровська область', 'Донецька область',
```
Comment (vitaliimak) [inline client/src/components/info/restaurants.js]: Don't use non Latin letters in code. If you need localization, use special tools for it

**Review by vitaliimak — CHANGES_REQUESTED** [review]

File: server/src/main/java/com/lunchtime/models/RestaurantVisit.java (hunk 1 above)
Comment (VozNick) [reply server/src/main/java/com/lunchtime/models/RestaurantVisit.java:1]: I mean it`s for sorting reason...
in restaurant list page, you can sort restaurants by your last visit. This class will keep dates of all rest. that you have visited...

Comment (vitaliimak) [reply server/src/main/java/com/lunchtime/models/RestaurantVisit.java:1]: Many users use this app.
Bob visited Restaurant A in 2020-03-05
Maria visited Restaurant A in 2020-03-06
Bob visited Restaurant B in 2020-03-06
...

Your model doesn't connect to Persons, how you prevent collisions between Persons? How you save date 2020-03-05 (Bob) and 2020-03-06 (Maria)? Does this functionality need?

**Pull request closed without merging**